manage/trigger/webhooks.rst

```diff
@@ -0,0 +1,365 @@
+Webhooks
+========
+
+Webhooks are a way to integrate Zammad with other web services or applications,
+allowing them to subscribe to live updates about new tickets
+instead of having to poll the Zammad server every *n* minutes.
+
+.. note:: ⌛ **Webhooks may not arrive immediately.**
+
+   Webhooks are sent out with the same priority and order as email triggers.
+   If webhook dispatch fails (*e.g.,* because the receiving server is misconfigured),
+   Zammad will retry up to four times.
+
+How does it work?
+-----------------
+
+Under the hood, Zammad sends a POST request
+to a third-party URL (“API endpoint”) you specify in the New Trigger dialog.
+The application server behind this URL/endpoint
+must be configured to receive messages from Zammad
+and handle the attached data accordingly.
+
+Webhook requests from Zammad contain the following JSON data about new/incoming tickets:
+
+* ticket attributes/metadata
+* *all* associated articles
+* associated users (*e.g.,* article senders, owners, etc.)
+* associated user roles
+* associated user organizations (if applicable)
+* associated groups
+
+Request headers
+---------------
+
+Zammad sends the following headers in each webhook POST request:
+
+:``User-Agent``:        ``"Zammad User Agent"``
+:``X-Zammad-Trigger``:  The name of the originating trigger
+:``X-Zammad-Delivery``: A unique, random ID string
+:``X-Hub-Signature``:   The SHA-1 hash of your HMAC-SHA1 signature token
+                        (assuming you provided one when creating your trigger)
```

Comment:
I really don't like the styling of the attributes.
This is how it looks compiled:

![image](https://user-images.githubusercontent.com/6549061/99798356-2d664100-2b31-11eb-8bfb-6cc77722c462.png)


Personal feel: Super ugly. Beside of the `:` being were it should be?